xbmc/pvr/PVRGUIActions.cpp

```diff
@@ -700,6 +702,50 @@ namespace PVR
                                             CVariant{timer->Title()});
   }
 
+  class AsyncRecordingAction : private IRunnable
+  {
+  public:
+    bool Execute(const CFileItemPtr &item = CFileItemPtr());
```

Comment:
Yeah, it's a broken pattern because const ref is so ingrained people just continued with it. It makes no sense to use it on shared_ptr and actually can make things worse as it gives the impression that things are fine but the ref count isn't incremented so things can blow up unexpectedly.